src/main/java/woohakdong/server/common/config/SecurityConfig.java

```diff
@@ -1,19 +1,29 @@
 package woohakdong.server.common.config;
 
+import lombok.RequiredArgsConstructor;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.Customizer;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.http.SessionCreationPolicy;
 import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+import woohakdong.server.common.security.jwt.JWTFilter;
+import woohakdong.server.common.security.jwt.JWTUtil;
 
 @Configuration
 @EnableWebSecurity
+@RequiredArgsConstructor
 public class SecurityConfig {
+    private final JWTUtil jwtUtil;
```

Comment:
JwtFilter를 컴포넌트 처리하고, 
```suggestion
    private final JWTFilter jwtFilter;
```
위 부분처럼 변경하면 좋을 것 같아요.